docs/Reference/API-Methods.md

```diff
@@ -0,0 +1,2912 @@
+
+* `initiator`: *string* - contract extension initiator's Ethereum address
+
+* `recipient`: *string* - new participant's Ethereum address; the participant must later approve the extension using this address.
+
+* `recipientPtmKey`: *string* - new participant's Tessera public key
+
+!!! example
+
+    === "curl HTTP request"
+
+        ```bash
+        curl -X POST http://127.0.0.1:22000 --data '{"jsonrpc":"2.0","method":"quorumExtension_activeExtensionContracts","id":10}' --header "Content-Type: application/json"
+        ```
+
+    === "JSON result"
+
+        ```json
+        {
+          "jsonrpc":"2.0",
+          "id":10,
+          "result": [{
+            "managementContractAddress":"0xc4e9de0bd5e0a5fd55ef5d6f2b46eba930a694a3",
+            "contractExtended":"0x027692c7ebdc81c590250e615ab571a0d14eff2d",
+            "creationData":"Zvo1Rnrfq4phIJbzKObyCBWSXTbEJGPOq5+jDCWccnPpA7K6OvIssCMLJ54f32uuEeczeVNC46QMk52lCOWbtg==",
+            "initiator":"0xed9d02e382b34818e88b88a309c7fe71e65f419d",
+            "recipient":"0x0fbdc686b912d7722dc86510934589e0aaf3b55a",
+            "recipientPtmKey":"1iTZde/ndBHvzhcl7V68x44Vx7pl8nwx9LqnM/AfJUg="
+          }]
+        }
+        ```
+
+    === "geth console request"
+
+        ```javascript
+        quorumExtension.activeExtensionContracts
+        ```
+
+    === "geth console result"
+
+        ```json
+        [{
+          "managementContractAddress":"0xc4e9de0bd5e0a5fd55ef5d6f2b46eba930a694a3",
+          "contractExtended":"0x027692c7ebdc81c590250e615ab571a0d14eff2d",
+          "creationData":"Zvo1Rnrfq4phIJbzKObyCBWSXTbEJGPOq5+jDCWccnPpA7K6OvIssCMLJ54f32uuEeczeVNC46QMk52lCOWbtg==",
+          "initiator":"0xed9d02e382b34818e88b88a309c7fe71e65f419d",
+          "recipient":"0x0fbdc686b912d7722dc86510934589e0aaf3b55a",
+          "recipientPtmKey":"1iTZde/ndBHvzhcl7V68x44Vx7pl8nwx9LqnM/AfJUg="
+        }]
+        ```
+
+### `quorumExtension_getExtensionStatus`
+
+Retrieves the status of the specified contract extension.
+
+#### Parameters
+
+`managementContractAddress`: *string* - address of the extension management contract
+
+#### Returns
+
+`result`: *string* - status of contract extension (`ACTIVE` or `DONE`)
+
+!!! example
+
+    === "curl HTTP request"
+
+        ```bash
+        curl -X POST http://127.0.0.1:22000 --data '{"jsonrpc":"2.0","method":"quorumExtension_getExtensionStatus","params":["0x1349f3e1b8d71effb47b840594ff27da7e603d17"],"id":10}' --header "Content-Type: application/json"
+        ```
+
+    === "JSON result"
+
+        ```json
+        {
+          "jsonrpc":"2.0",
+          "id":10,
+          "result":"DONE"
+        }
+        ```
+
+    === "geth console request"
+
+        ```javascript
+        quorumExtension.getExtensionStatus("0x1349f3e1b8d71effb47b840594ff27da7e603d17")
+        ```
+
+    === "geth console result"
+
+        ```json
+        "DONE"
+        ```
+
+## Debug methods
```

Comment:
It seems that we forgot to doc `debug_privateStateRoot` (see ConsenSys/quorum#1148) so maybe this is the right moment to add it here.